internal/envoy/v3/bootstrap.go

```diff
@@ -156,6 +162,27 @@ func bootstrap(c *envoy.BootstrapConfig) ([]bootstrapf, error) {
 
 func bootstrapConfig(c *envoy.BootstrapConfig) *envoy_bootstrap_v3.Bootstrap {
 	return &envoy_bootstrap_v3.Bootstrap{
+		LayeredRuntime: &envoy_bootstrap_v3.LayeredRuntime{
+			Layers: []*envoy_bootstrap_v3.RuntimeLayer{
+				{
+					Name: "base",
+					LayerSpecifier: &envoy_bootstrap_v3.RuntimeLayer_StaticLayer{
+						StaticLayer: &structpb.Struct{
+							Fields: map[string]*structpb.Value{
+								"re2.max_program_size.error_level": {Kind: &structpb.Value_NumberValue{NumberValue: maxRegexProgramSizeError}},
+								"re2.max_program_size.warn_level":  {Kind: &structpb.Value_NumberValue{NumberValue: maxRegexProgramSizeWarn}},
+							},
+						},
+					},
+				},
+				{
+					Name: "admin",
+					LayerSpecifier: &envoy_bootstrap_v3.RuntimeLayer_AdminLayer_{
+						AdminLayer: &envoy_bootstrap_v3.RuntimeLayer_AdminLayer{},
+					},
```

Comment:
Does this layer need to be specified?

Reply:
Yeah, by default if you dont specify any runtime layers, Envoy inserts a default admin layer which should let you modify runtime values via the admin endpoint. If we omit it and specify other layers, you lose that functionality. (see https://www.envoyproxy.io/docs/envoy/latest/configuration/operations/runtime#admin-console)

Reply:
I can add a comment to that effect ^

Reply:
Perfect, thanks!